examples/web-pages-docker.ps1

```diff
@@ -1,14 +1,17 @@
-Import-Module /usr/local/share/powershell/Modules/Pode/Pode.psm1 -Force -ErrorAction Stop
+try {
+    Import-Module /usr/local/share/powershell/Modules/Pode/Pode.psm1 -Force -ErrorAction Stop
+}
+catch { throw }
 
 <#
 docker-compose up --force-recreate --build
 #>
 
-# create a server, and start listening on port 8085
+# create a server, and start listening on port 8081
 Start-PodeServer -Threads 2 {
 
-    # listen on *:8085
-    Add-PodeEndpoint -Address * -Port 8085 -Protocol Http
+    # listen on *:8081
+    Add-PodeEndpoint -Address * -Port 8081 -Protocol Http
```

Comment:
There's a `docker-compose.yml` and a `Dockerfile` in the examples folder which will need ports updating to 8081 as well :)